packages/x-date-pickers/src/CalendarPicker/DayPicker.test.tsx

```diff
@@ -0,0 +1,56 @@
+import * as React from 'react';
+import { expect } from 'chai';
+import { spy } from 'sinon';
+import { userEvent, screen } from '@mui/monorepo/test/utils';
+import { DayPicker } from './DayPicker';
+import { adapterToUse, createPickerRenderer } from '../../../../test/utils/pickers-utils';
+
+describe('<DayPicker />', () => {
```

Comment:
I added the test directly on `DayPicker`, I think the structure of the pickers favors more unit tests on the smaller components.